scripts/docker/ros2/ros2_rolling_deb.Dockerfile

```diff
@@ -0,0 +1,13 @@
+
+ARG UBUNTU_VERSION=16.04
```

Comment:
I guess you are used to the `UBUNTU_VERSION` default value not mattering because it can be overridden from the wrapper script, but in this case I think choosing the value `16.04` is confusing because it is not one of the valid versions to use, correct? (Same comment applies to some other files.)